index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Augustin Anders</title>
+  </head>
+  <body>
+    <header>
+      <h1>Augustin Anders</h1>
+      <nav>
+        <button type="button">portfolio</button>
+        <button type="button">about</button>
+        <button type="button">contact</button>
+      </nav>
```

Comment:
I think the buttons have no real navigation purpose at the moment. You might want to add href="#id" to create a working navigation.